docs/src/common-collections/storage_map.md

```diff
@@ -34,9 +34,12 @@ To insert key-value pairs into a storage map, we can use the `insert` method, as
 
 Note two details here. First, in order to use `insert`, we need to first access the storage map using the `storage` keyword. Second, because `insert` requires _writing_ into storage, a `#[storage(write)]` annotation is required on the ABI function that calls `insert`.
 
-> **Note**: The storage annotation is also required for any private function defined in the contract that tries to insert into the map.
+> **Note**
```

Comment:
Does the linter not like the colon after `**Note**`? Without it, the note looks a bit weird:
![image](https://user-images.githubusercontent.com/59666792/184643427-c091acbc-4048-4f32-912b-cc4940c402ef.png)
Looking at the Rust book, notes that start with `Note` do have a colon.

Reply:
Note strictly, no. It's to make it look like this on GitHub

![note](https://user-images.githubusercontent.com/3290375/184644474-d7d34327-2df2-45c3-a8c9-0a69ab1a4305.png)

and the above screenshot on Mdbook.

Reply:
But shouldn't we care more about how it renders on Mdbook? It's a shame that book does not render notes the same was as Github 😞

Reply:
The book still renders them as a box, instead of just quote text, with the changed format.

Reply:
Yeah, I am just more concerned with the lack of the colon (`:`) after `Note` which makes it look weird IMO, but maybe the fact that `Note` is in bold make that acceptable.